presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizers.java

```diff
@@ -186,6 +186,20 @@ public PlanOptimizers(
                 taskCountEstimator);
     }
 
+    @PostConstruct
+    public void initialize()
```

Comment:
this should go in the previous commit (undoing the accidental formatting changes you made there).

Reply:
there's an issue to move these two methods, fixed